cesium/tests/test_featureset.py

```diff
@@ -0,0 +1,34 @@
+import os
+from os.path import join as pjoin
+import numpy as np
+import numpy.testing as npt
+import xarray as xr
+from cesium.featureset import Featureset
+
+
+DATA_PATH = pjoin(os.path.dirname(__file__), "data")
+
+
+def test_repr():
+    """Testing Featureset printing."""
+    fset = Featureset(xr.open_dataset(pjoin(DATA_PATH, "test_featureset.nc")))
+    repr(fset)
+
+
+def test_impute():
```

Comment:
Shouldn't we be testing the other impute strategies? At least to catch any API/function signature changes on the part of `sklearn`.